src/main/resources/db/changelog/changelog-master.xml

```diff
@@ -5,5 +5,9 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
 
-    <include file="db/changelog/changelog-1.0.xml"/>
+    <include file="db/changelog/changelog-1.0.sql"/>
```

Comment:
As a "best practice" suggestion usually name files to signify their role/action. In this case create_table_products.sql for example. Also You sohuld group file incudes as a "changeset" ex.:https://docs.liquibase.com/concepts/basic/changeset.html